src/pydolphinscheduler/tasks/test_http_conversion.py

```diff
@@ -0,0 +1,32 @@
+from pydolphinscheduler.core.parameter import ParameterHelper, ParameterType
+from http.py import Http  # Replace 'your_module_path' with the actual path or import structure
+
+def test_http_params_conversion():
```

Comment:
Great! you add some tests for this, but we already have tests for task http in https://github.com/apache/dolphinscheduler-sdk-python/blob/f82e07d232380fe9f5582db4dcca920ae00befd4/tests/tasks/test_http.py could you merge into that file?